CROSSSECTOR/DIH/DigitalInnovationHubService/schema.json

```diff
@@ -0,0 +1,138 @@
+		},
+		{
+			"properties": {
+				"serviceImage": {
+					"type": "object",
+					"description": "Property. [StructuredValue](https://schema.org/StructuredValue). Image of the Digital Innovation Hub Service.",
+					"properties": {
+						"base64Data": {
+							"type": "string"
+						},
+						"filename": {
+							"type": "string"
+						}
+					},
+					"required": [
+						"base64Data",
+						"filename"
+					]
+				},
+				"category": {
+					"type": "string",
+					"description": "Property. [Text](https://schema.org/Text). Category of the Digital Innovation Hub Service."
+				},
+				"serviceType": {
+					"type": "string",
+					"description": "Property. [Text](https://schema.org/Text). ServiceType of the Digital Innovation Hub Service."
+				},
+				"service": {
+					"type": "string",
+					"description": "Property. [Text](https://schema.org/Text). Service of the Digital Innovation Hub Service."
+				},
+				"target": {
+					"type": "array",
+					"description": "Property. [StructuredValue](https://schema.org/StructuredValue). Targets of the Digital Innovation Hub Service.",
+					"items": [{
+							"type": "string"
+						},
+						{
+							"type": "string"
+						}
+					]
+				},
+				"url": {
+					"type": "string",
+					"description": "Property. [URL](https://schema.org/URL). URL of the Digital Innovation Hub Service."
+				},
+				"additionalmaterial": {
+					"type": "array",
+					"description": "Property. [StructuredValue](https://schema.org/StructuredValue). Additional Materials of the Digital Innovation Hub.",
+					"items": [{
+						"type": "object",
+						"properties": {
+							"label": {
+								"type": "string"
+							},
+							"url": {
+								"type": "string"
+							}
+						},
+						"required": [
+							"label",
+							"url"
+						]
+					}]
+				},
+				"contacts": {
+					"type": "array",
+					"description": "Property. [StructuredValue](https://schema.org/StructuredValue). Contacts of the Digital Innovation Hub.",
+					"items": [{
+						"type": "object",
+						"properties": {
+							"name": {
+								"type": "string"
+							},
+							"email": {
+								"type": "string"
+							}
+						},
+						"required": [
+							"name",
+							"email"
+						]
+					}]
+				},
+				"author": {
+					"type": "string",
+					"description": "Property. [Text](https://schema.org/Text). Author of the Digital Innovation Hub Service."
+				},
+				"relation": {
+					"type": "object",
+					"description": "Property. [StructuredValue](https://schema.org/StructuredValue). Relation of the Digital Innovation Hub.",
+					"properties": {
+						"dih": {
+							"type": "array",
+							"items": [{
+								"type": "object",
+								"properties": {
+									"to": {
+										"type": "string"
+									}
+								},
+								"required": [
+									"to"
+								]
+							}]
+						}
+					},
+					"required": [
+						"dih"
+					]
+				},
+				"extra": {

```

Comment:
not much fond of 'open' objects.  (without defined subattributes)
if you do not have anything clear then remove it. (and removed it as well from the examples)
It is not a big deal to add attributes later if necessary.